interface/xtreemfs/MRC.proto

```diff
@@ -527,6 +527,11 @@ message xtreemfs_replica_addRequest {
   required Replica new_replica = 2;
 }
 
+message xtreemfs_replica_addResponse {
```

Comment:
Please add brief descriptions of new messages and operations (I know it's even incomplete for the existing).